### DISCUSSION TITLE
能不能实现输入候选以横排显示，输入方案菜单以竖排显示呢

### DISCUSSION THREAD

**Discussion by quantumfallout** [rime/home#1280] [Q&A]
![Snipaste_2023-04-29_23-19-45](https://user-images.githubusercontent.com/16105768/235310316-c33848ed-3b4c-4b19-8969-b34f1be1d585.png)
有时输入方案的名字长起来，横排看着挺累的

**Comment by LEOYoon-Tsaw** [comment]
能，需要編程

**Comment by pdog18** [comment]
> ![Snipaste_2023-04-29_23-19-45](https://user-images.githubusercontent.com/16105768/235310316-c33848ed-3b4c-4b19-8969-b34f1be1d585.png) 有时输入方案的名字长起来，横排看着挺累的

真的存在这种经常在不同方案之间切换的情况吗? 为什么呢? 看你有拼音和粤语拼音 还有 esay-en, 您平时使用输入法的过程中, 经常需要在这三个方案中切换吗?
